### PR TITLE
[BPK-434] always show sort icons

### PR DESCRIPTION
**Pull request by janeklb**
Addressing [comments from Kartikey](https://gojira.skyscanner.net/browse/BPK-434?focusedCommentId=1174220&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-1174220)
- always show sort icons
- change sort icon colour

jan-04-2018 21-29-39

### CODE REVIEW

File: packages/bpk-component-datatable/src/bpk-data-table-column.scss [hunk 1]
```diff
@@ -33,11 +41,25 @@
...
+  &:hover {
+    fill: $bpk-color-gray-500;
+  }
+  &--selected {
+    &, &:hover { fill: $bpk-color-blue-700 };
+  }
+}
+
+.bpk-data-table-column__sort-icon--up {
+  margin-bottom: -6px;
```
Comment (janeklb) [inline packages/bpk-component-datatable/src/bpk-data-table-column.scss]: I'm not sure if this is the right way of specifying such a custom value.. maybe there's a better css solution out there too.

**Comment by janeklb** [comment]
found a little bug, marked as `[WIP]`

**Comment by janeklb** [comment]
pushed fix

File: packages/bpk-component-datatable/src/bpk-data-table-column.scss [hunk 2]
```diff
@@ -33,11 +42,30 @@
...
+  &--selected {
+    &,
+    &:hover {
+      fill: $bpk-color-blue-700;
+    }
+  }
+}
+
+.bpk-data-table-column__sort-icon--up {
+  margin-bottom: -($bpk-border-size-xl * 2);
```
Comment (janeklb) [inline packages/bpk-component-datatable/src/bpk-data-table-column.scss]: Reusing a border mixin/bond doesn't really make sense here. What's the best way of solving this? Maybe there's a better approach (the purpose of this is to reduce the vertical spacing between the up and down icons).

Comment (shaundon) [reply packages/bpk-component-datatable/src/bpk-data-table-column.scss]: You could use `spacing-xs`, which is 6px, so you wouldn't have to do the `* 2` part. I can't really think of a better way to do it than how you have though, other than that.

Comment (janeklb) [reply packages/bpk-component-datatable/src/bpk-data-table-column.scss]: Sounds good - cheers :)

**Pull request merged**